reference/speaker.md

```diff
@@ -60,106 +60,61 @@ It is possible to stop all the sounds currently playing in a speaker by setting
 
 **Name**
 
-**wb\_speaker\_set\_language**, **wb\_speaker\_get\_language**, **wb\_speaker\_speak** - *perform text-to-speech*
+**wb\_speaker\_set\_engine**, **wb\_speaker\_set\_language**, **wb\_speaker\_get\_engine**, **wb\_speaker\_get\_language**, **wb\_speaker\_speak** - *perform text-to-speech*
 
 {[C++](cpp-api.md#cpp_speaker)}, {[Java](java-api.md#java_speaker)}, {[Python](python-api.md#python_speaker)}, {[Matlab](matlab-api.md#matlab_speaker)}, {[ROS](ros-api.md)}
 
 ```c
 #include <webots/speaker.h>
 
-void wb_speaker_set_language(WbDeviceTag tag, const char *language);
-const char * wb_speaker_get_language(WbDeviceTag tag);
+bool wb_speaker_set_engine(WbDeviceTag tag, const char *engine);
+bool wb_speaker_set_language(WbDeviceTag tag, const char *language);
+const char *wb_speaker_get_engine(WbDeviceTag tag);
+const char *wb_speaker_get_language(WbDeviceTag tag);
 void wb_speaker_speak(WbDeviceTag tag, const char *text, double volume);
 ```
 
 **Description**
 
-The `wb_speaker_set_language` function allows the user to set the language of the text-to-speech engine. The `language` parameter should be set to one of the following values:
+The `wb_speaker_set_engine` function allows the user to set the text-to-speech engine that is going to be used by a speaker. The `engine` parameter should be one of the following values:
 
-  - `"en-US"` for American English (default value)
-  - `"en-UK"` for British English
-  - `"de-DE"` for German
-  - `"es-ES"` for Spanish
-  - `"fr-FR"` for French
-  - `"it-IT"` for Italian
+  - `"pico"` for the SVOX Pico text-to-speech engine (default value).
+  - `"microsoft"` for the Microsoft SAPI5 text-to-speech engine (only available on Windows).
 
-The `wb_speaker_get_language` function allows the user to get the language of the text-to-speech.
+The function returns `false` if the engine cannot not be set and `true` otherwise.
 
-The `wb_speaker_speak` function allows the user to execute text-to-speech on the speaker. The value of the `text` parameter is converted into sound by Webots using the language specified by the `wb_speaker_set_language` function. The resulting sound is played through the speaker. The specified text could be plain text including punctuation signs such as "Hello world!", or can be enriched with special effects to make it more realistic. Such effects are speficied with XML tags.
+The `wb_speaker_get_engine` function allows the user to get the engine of the text-to-speech for a speaker device.
```

Comment:
I'd say "get the text-to-speech engine" instead of "get the engine of the text-to-speech"